db/data/20190418114011_initialise_state_for_users.rb

```diff
@@ -0,0 +1,15 @@
+class InitialiseStateForUsers < ActiveRecord::Migration[5.2]
+  def up
+    User.find_each do |user|
```

Comment:
Could just be a oneliner `User.find_each(&:save)`

Reply:
Done already by rubocop